include/oneapi/dpl/pstl/hetero/dpcpp/utils_ranges_sycl.h

```diff
@@ -118,6 +118,18 @@ struct all_view_fn
     }
 };
 
+#if _ONEDPL_SYCL_PLACEHOLDER_HOST_ACCESSOR_DEPRECATED
+struct all_host_view_fn
```

Comment:
I compared the public API of `host_all` before and after the change:

  | before | after
-- | -- | --
value_type | + | +
begin | + | +
end | + | +
size | + | +
empty | + | +
require_access | + | -
constructor(buffer, offset, n) | + | -
constructor(accessor) | + | -
noop (shall return the same view) | + | -


Could you make additional changes in order not to break backward compatibility?